liquibase-service/src/main/resources/db/changelog/db.rdb.changelog-16.1.yaml

```diff
@@ -12,4 +12,32 @@ databaseChangeLog:
       changes:
         - sqlFile:
             path: 001-create_data_sync_config-001.sql
+            splitStatements: false
+  - changeSet:
+      id: 2
+      author: liquibase
+      changes:
+        - sqlFile:
+            path: 001-create_data_sync_config-002.sql
```

Comment:
The file path is incorrect. From id:2,  It should be  like this: 001-generate_data_for_data_sync_config-001.sql
id3: path: 001-generate_data_for_data_sync_config-002.sql

Each of the database scripts should be represented in the changelog if we require liquibase to run them.